builtin-functions/_functions.txt

```diff
@@ -1614,3 +1614,5 @@ class DateTimeImmutable implements DateTimeInterface {
   public function getOffset(): int;
   public function getTimestamp(): int;
 }
+
+function getenv(string $varname = '', bool $local_only = false): mixed;
```

Comment:
Why do you define return type as `mixed`, meanwhile the return type is `string | false` aka `Optional<String>`?
Why don't you add `getenv() : array` to this file?
Could explain these moments, please?

Reply:
Because I can't overload function in php.